src/main/scala/org/xalgorithms/apps/KafkaStreamingApplication.scala

```diff
@@ -13,21 +16,31 @@ import org.apache.spark.streaming.Seconds
 import org.apache.spark.streaming.StreamingContext
 import org.apache.spark.streaming.dstream.DStream
 import org.apache.spark.streaming.kafka.KafkaUtils
+import org.bson.Document
 
-class KafkaStreamingApplication(cfg: ApplicationConfig) {
+class KafkaStreamingApplication(cfg: ApplicationConfig) extends Serializable {
```

Comment:
Why should the core application class be serializable? If there's a critical reason, add a comment.